src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -464,6 +464,18 @@ const analyticsEvents = {
       action: 'change',
       event: 'int-radio-button-option-click',
       prefix: 'radio-button',
+      ga4: {
+        event: 'interaction',
+        component_name: 'va-radio',
+        custom_string_1: 'component_library',
+      },
+      /* Component to GA4 parameters */
+      mapping: {
+        'radio-button-label': 'heading_1',
+        'radio-button-optionLabel': 'value',
+        'radio-button-required': 'required',
+        version: 'component_library',
```

Comment:
Even though we aren't really passing a component_version, this mapping has been getting set to `version: 'component_version',` for the others.